code/src/Pages/MovieDetails.js

```diff
@@ -0,0 +1,38 @@
+import React, {useEffect, useState } from "react"
+import { useParams, Link } from "react-router-dom"
+
+import "./MovieDetails.css"
+
+export const MovieDetails = () => {
+  const { movieId } = useParams()
+  const [movies, setMovies] = useState([])
+
+  useEffect(() => {
+    fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=10f97b185c42758de306cf613370d0c8&language=en-US`)
+      .then((res) => res.json())
+      .then((json) => {
+          setMovies(json)
+      })
+  }, [movieId])
+
+  return (
+    <section className="detail-section">        
+      <div className="go-back">
+        <Link to={`/`}>
```

Comment:
we don't need to use backticks (``) over here, single or double quotes ('' / "") are enough